pkg/checkpoint/process_test.go

```diff
@@ -264,8 +266,8 @@ func TestProcess(t *testing.T) {
 				},
 				"AA": {},
 			},
-			expectStop:        []string{"AA", "kube-system/pod-checkpointer"},
-			expectGraceRemove: []string{"AA", "kube-system/pod-checkpointer"},
+			expectStart:       []string{"kube-system/pod-checkpointer", "BB"},
```

Comment:
Same as above, the checkpointer stays active during the grace period, and since BB is not locally running we also start its checkpoint as expected (and then promptly remove it).

Reply:
Hmm, it's not running locally, but its parent is -- so we shouldn't start it, right?

Reply:
I thought localRunning (from the CRI shim) is the source of truth? The old code didn't actually look at localParent for anything actionable, my understanding is that it's just a cached mirror of apiParents.

Reply:
Ignore me. We added test case to old code and this is the same behavior